Directory.Build.props

```diff
@@ -6,6 +6,7 @@
     <PackageLicenseExpression Condition=" '$(PackAsTool)' != 'true' ">Apache-2.0</PackageLicenseExpression>
     <NoWarn>$(NoWarn);3186,0042</NoWarn><!-- circumvent an error with the fake dependencymanager for paket: https://github.com/dotnet/fsharp/issues/8678 -->
     <NoWarn>$(NoWarn);NU1902</NoWarn><!-- NU1902 - package vulnerability detected -->
+    <WarnOn>1182;3390;$(WarnOn)</WarnOn>
```

Comment:
Can you split the new one out and add a comment for what this code maps to? I can never keep these straight in my head.

Reply:
Sure, although I'm not quite sure what you mean by splitting them out.

Reply:
exactly what you ended up doing - separate each NoWarn onto a separate line so it can be commented more easily. thanks!